### PR TITLE
doc: add faq about missing heartbeat

### PR DESCRIPTION
**Pull request by xiang90**
Fix https://github.com/coreos/etcd/issues/6955

/cc @gyuho @heyitsanthony

### CODE REVIEW

File: Documentation/faq.md [hunk 1]
```diff
@@ -62,8 +62,22 @@ Expensive user requests which access too many keys (e.g., fetching the entire ke
...
+
+Usually this issue is caused by a slow disk. Before the leader sends heartbeats attached with metadata, it may need to persist the metadata to disk. The disk could be experiencing contention among etcd and other applications, or the disk is too simply slow (e.g., a shared virtualized disk). To rule out a slow disk from causing this warning, monitor  [wal_fsync_duration_seconds](wal_fsync_duration_seconds) (p99 duration should be less than 10ms) to confirm the disk is reasonably fast. If the disk is too slow, assigning a dedicated disk to etcd or using faster disk will typically solve the problem.
```
Comment (heyitsanthony) [inline Documentation/faq.md]: broken link?

File: Documentation/faq.md [hunk 2]
```diff
@@ -62,8 +62,22 @@ Expensive user requests which access too many keys (e.g., fetching the entire ke
...
+
+The second most common cause is CPU starvation. If monitoring of the machine’s CPU usage shows heavy utilization, there may not be enough compute capacity for etcd. Moving etcd to dedicated machine, increasing process resource isolation  with cgroups, or renicing the etcd server process into a higher priority can usually solve the problem.
+
+A slow network can also cause this issue. If network metrics among the etcd machines shows long latencies or high drop rate, there may not be enough network capacity for etcd. Moving etcd members to a less congested network will typically solve the problem. However, if the etcd cluster is deployed across data centers, long latency between members is expected. For such deployments, tune the `heartbeat-interval` configuration to roughly match the round trip time between the machines, and the `election-timeout` configuration to be at least 5 * `heartbeat-interval`. See (tuning documentation)[tuning] for detailed information.
```
Comment (heyitsanthony) [inline Documentation/faq.md]: broken link?

**Comment by xiang90** [comment]
@heyitsanthony All fixed.

**Comment by heyitsanthony** [comment]
lgtm

**Comment by xiang90** [comment]
@heyitsanthony Can you approve this pr?

**Pull request merged**